src/middlewared/middlewared/api/base/decorator.py

```diff
@@ -19,6 +19,8 @@ def api_method(
     rate_limit=True,
     roles: list[str] | None = None,
     private: bool = False,
+    no_auth_required: bool = False,
```

Comment:
Should we avoid double negation with `auth_required: bool = True`?

Reply:
That's not a bad idea.